src/app/analyze/stack/stack-details/stack-details.component.ts

```diff
@@ -346,21 +358,22 @@ export class StackDetailsComponent implements OnInit {
       );
   }
 
-  handleNext(value: any): void {
+  private handleNext(value: any): void {
     // ++ this.currentIndex;
     // Hit a new Ajax call and populate the Array
-    let nextObservable: Observable<any> = this.renderNextService.getNextList(this.recoArray[this.currentIndex]['url']);
+    let nextObservable: Observable<any> 
+    = this.renderNextService.getNextList(this.recoArray[this.currentIndex]['url']);
```

Comment:
I would rather have this on one line then to break at the equals. The line length is a suggestion. I've been thinking about increasing it to 120. 
If you want to break it, then do it at a period and indent to that place.